src/Button/Button.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import propTypes from 'prop-types'
+
+/**
+ * Primary UI component for user interaction
+ */
+export const Button = ({ className, children, onClick, type, ref, ...props }) => {
```

Comment:
Start of a button component. I suspect we will have a number of levers (big, etc) later.